### DISCUSSION TITLE
User-Defined Functions no longer link to the source file?

### DISCUSSION THREAD

**Discussion by csnydermph** [posit-dev/positron#4339] [Feedback]
In RStudio, selecting a function from the environment viewer would open up the file where the function was defined, which made editing  / managing user-defined functions very managable.

In Positron, currently I don't see this functionality.
Will we be able to navigate to the source files in this way in the future, or is that only going to be a feature of RStudio?

**Comment by jmcphers** [comment]
It's totally possible to add this to Positron! I've filed your feedback as an issue here: https://github.com/posit-dev/positron/issues/4341
